beacon_node/http_api/tests/tests.rs

```diff
@@ -2410,12 +2410,16 @@ impl ApiTester {
                 sk.sign(message).into()
             };
 
-            let block = self
+            let block_bytes = self
                 .client
-                .get_validator_blocks::<E, FullPayload<E>>(slot, &randao_reveal, None)
+                .get_validator_blocks_ssz::<E, FullPayload<E>>(slot, &randao_reveal, None)
                 .await
                 .unwrap()
-                .data;
+                .expect("block bytes");
+
+            let block =
+                BeaconBlock::<E, FullPayload<E>>::from_ssz_bytes(&block_bytes, &self.chain.spec)
+                    .expect("block bytes can be decoded");
```

Comment:
I think it would be better to test this separately, like what you did with `test_blinded_block_production_ssz` - it's more explicit and less likely to be modified by accident in the furture - perhaps just one similar to the `block_production` test will do?